flow-report/test/App-test.tsx

```diff
@@ -0,0 +1,15 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {App} from '../App';
+import {render} from '@testing-library/preact';
+
+const flow = require('../../lighthouse-core/test/fixtures/fraggle-rock/reports/sample-lhrs.json');
+
+it('Renders a standalone report', () => {
+  const root = render(<App flow={flow}/>);
+  expect(root.baseElement.outerHTML).toMatchInlineSnapshot(`"<body><div><select><option value=\\"0\\">[2021-08-03T18:28:13.296Z] [navigation] https://www.mikescerealshack.co/</option><option value=\\"1\\">[2021-08-03T18:28:31.789Z] [timespan] https://www.mikescerealshack.co/search?q=call+of+duty</option><option value=\\"2\\">[2021-08-03T18:28:36.856Z] [snapshot] https://www.mikescerealshack.co/search?q=call+of+duty</option></select><div><div><h1>https://www.mikescerealshack.co/</h1><h2>performance: 0.99</h2><h2>accessibility: 1</h2><h2>best-practices: 1</h2><h2>seo: 1</h2><h2>pwa: 0.3</h2></div></div></div></body>"`);
```

Comment:
want to narrow the scope of this and add one more assertion for the selection change? I'm thinking some structure that will be useful beyond this PR even if the assertions change :)

psuedo-code below, probably won't work exactly like this, but something similar

```js
it('renders all the reports in the flow', async () => {
  render(<App flow={flow}/>);

  await screen.findByText(/navigation/)
  await screen.findByText(/timespan/)
  await screen.findByText(/snapshot/)
})

it('renders a particular report on selection', async () => {
  render(<App flow={flow}/>);

  getByText('performance: 0.99')
  fireEvent.change(getByRole('select'), {target: {value: '2'}});
  await findByText('performance: <some other number>')
})
```

Reply:
Spoiler alert, the select box isn't going to stick around very long. I didn't want to spend time testing components that are going to get overhauled in the next patch.

This test will definitely have more specific assertions in the future. For the shell report, I wanted something that just ensures something gets rendered.

I think I'll change the first test for now, but probably won't add the second.

Reply:
ya for sure, `it.todo` wfm for the next one :)